docs/attending/workshops.md

```diff
@@ -7,4 +7,4 @@ Common Workshops:
 * Basic Python Programming
 * Making your first website
 
-MLH events will likely do something from this [list](https://localhost.mlh.io/#activities).
+MLH events will do something from this [list](https://localhost.mlh.io/#activities).
```

Comment:
For clarity – can we update this line to something like this?

> MLH Member Events will run a [BlackRock-branded Capture the Flag](https://challenges.mlh.io/).

Reply:
Do they have to run a CTF? We didn't get one at HS?

Reply:
Yeah, as of this week, MLH will run a CTF rather than !Light at European events

Reply:
On top of other games/workshops or exclusively?

Reply:
By default, the Coach has to run the BlackRock CTF at every European MLH Member Event. If the organisers want the Coach to run something else like cup-stacking, then they can do that as well (if the organisers provide the cups) – but the Coach has to run the CTF at a minimum.

Reply:
Added that to the text